src/e2e-test/config/testcases-hl7v2.js

```diff
@@ -0,0 +1,13 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+// design the test cases
+var ruleSets = require('./rule-sets'); 
+var hl7v2Testcases = [
+    { dataFile: 'ADT01-28.hl7', templateFile: 'ORU_R01.hbs', testRules: ruleSets.commonRules}
```

Comment:
I feel like every top level template should have at least one testcase for it. As this PR is setting up the framework I see adding more tests as a separate work item, but I think that work item is needed.

Reply:
Yes, I agree with you. However, because some templates have errors now like 'ADT_A01.hbs', npm test will fail if it is added at present.